### Instructions

We will have another simple example on two sample t test (pooled- when the variances are equal). But this time this is a one sided t-test.

In a packing plant, a machine packs cartons with jars. It is supposed that a new machine will pack faster on the average than the machine currently used. To test that hypothesis, the times it takes each machine to pack ten cartons are recorded. The results, in seconds, are shown in the tables in the file `files_for_lab/machine.txt`. Assume that there is sufficient evidence to conduct the t test, does the data provide sufficient evidence to show if one machine is better than the other.

### Hypothesis Test

H0: The new machine does not pack faster on average than the current machine. μ_new ≥ μ_current <br>
HA: The new machine packs faster on average than the current machine. μ_new < μ_current

Further definitions:
- Since we are comparing the mean of one sample to a known population mean, we can use a two-sample t-test (one-tailed).
- We conduct a t-test.
- Level of significance alpha = 0.05.

In [24]:
import numpy as np
from scipy.stats import ttest_ind

old_machine = [42.1, 41, 41.3, 41.8, 42.4, 42.8, 43.2, 42.3, 41.8, 42.7]
new_machine = [42.7, 43.6, 43.8, 43.3, 42.5, 43.5, 43.1, 41.7, 44, 44.1]

t_statistic, p_value = ttest_ind(new_machine, old_machine)

print("T-statistic:", t_statistic)
print("P-value:", p_value)

T-statistic: 3.3972307061176026
P-value: 0.0032111425007745158


We compare the test statistic with the critical value or p-value to make a decision. Checking the critical value. <br>
dof = 10 <br>
alpha = 0.05 <br>
critical value = 2.228 <br>

To make a decision, we compare the obtained p-value with the chosen significance level (α). If the p-value is less than α, we reject the null hypothesis. If the p-value is greater than or equal to α, we fail to reject the null hypothesis.

**Conclusions**

The critical value is approx. (-)2.228. <br>
Since the p-value (0.0032) is smaller than the significance level (0.01), we can conclude that there is sufficient evidence to reject the null hypothesis. The observed test statistic is statistically significant. <br>
In this case, the test statistic (3.3972) is greater than the critical value (2.228). It suggests that the observed data is unlikely to occur if the null hypothesis is true. We reject it.
In relation to their average packaging speed the new machine packs fast than the old one.